addons/point_of_sale/tests/test_report_session.py

```diff
@@ -264,36 +264,3 @@ def test_report_bank_expected_different_than_counted(self):
             bank_payment_method_diffs={self.bank_pm1.id: -20})
         report = self.env['report.point_of_sale.report_saledetails'].get_sale_details(session_ids=[session1_id])
         self.assertEqual(report['payments'][1]['money_difference'], -20)
-
-        self.bank_pm1.outstanding_account_id = False
```

Comment:
This part is not relevant because pos payment method always have an outstanding account for bank